packages/table/components/CheckboxTable.tsx

```diff
@@ -154,7 +154,11 @@ class CheckboxTable extends React.PureComponent<
     };
 
     return (
-      <div ref={this.checkboxRef} className={display("inline-block")}>
+      <div
```

Comment:
I can't see the data-cy for this element.

Reply:
Really? I do
<img width="591" alt="Screen Shot 2019-07-19 at 10 44 40 AM" src="https://user-images.githubusercontent.com/2313998/61543856-4416f800-aa12-11e9-906d-f68a8d2654c3.png">

Reply:
Which element did you select?
![Screenshot from 2019-07-19 17-51-59](https://user-images.githubusercontent.com/40791275/61544422-18b1fe80-aa4e-11e9-94e4-c22a413b6f5e.png)

Reply:
Sorry, my screenshots weren't very helpful :x

<img width="612" alt="Screen Shot 2019-07-19 at 10 58 12 AM" src="https://user-images.githubusercontent.com/2313998/61544817-2d71a080-aa14-11e9-9ebf-db0b7fa3ab14.png">

That `data-cy` is just for the checkbox in the table header

Reply:
What if we want a table row?

Reply:
You could use the checkbox id or select using the `data-cy` attributes on the parent cell, but I've added a data-cy attribute specifically for checkbox cells for simplicity: `table-checkboxCell table-checkboxCell.row${rowIndex}`